### PR TITLE
Add sample swap view to Dashi

### PR DESCRIPTION
**Pull request by slazicoicr**

### CODE REVIEW

File: application/dash_application/views/sample_swaps.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    COL.LibraryRight,
+    special_cols["latest_run"],
+    COL.Result,
+    COL.LODScore,
+]
+
+# Pair-wise comparison is done within project (for now), so left project is sufficient
+ALL_PROJECTS = df_manipulation.unique_set(swap, PINERY_COL.StudyTitle)
+
+INITITIAL = {
```
Comment (avarsava) [inline application/dash_application/views/sample_swaps.py]: Inititial -> Initial?

File: application/dash_application/utility/sidebar_utils.py [hunk 2]
```diff
@@ -340,7 +340,7 @@ def get_requested_run_date_range(last_string) -> List[str]:
         return [None, None]
 
 
-def parse_query(query) -> List[str]:
+def parse_query(query) -> Dict[str, Any]:
```
Comment (avarsava) [inline application/dash_application/utility/sidebar_utils.py:343]: Why the change in signature?

Comment (slazicoicr) [reply application/dash_application/utility/sidebar_utils.py:343]: It was the wrong signature. `queries` is being returned and its a `Dict`

File: application/dash_application/views/sample_swaps.py (hunk 1 above)
Comment (slazicoicr) [reply application/dash_application/views/sample_swaps.py]: Yep.

**Review by avarsava — APPROVED** [review]

**Review by callunity — APPROVED** [review]
